project/src/PreloadedScripts/Settings.gd

```diff
@@ -12,6 +12,7 @@ const MIN_VOLUME := -70.0
 var master_volume := 0.8 setget set_master_volume
 var muted := false setget set_muted
 
+var level : int = -1 setget set_level
```

Comment:
Nie wiem czy nie zapisać tego w osobnym pliku. Możliwe że będziemy chceli więcej rzeczy zapisywać niż tylko poziom. Jak teraz sobie przygotujemy to potem będzie łatwo poszerzać.